hermeto/core/models/sbom.py

```
@@ -8,19 +8,19 @@
 from dataclasses import dataclass, field
 from datetime import datetime, timezone
 from functools import cached_property, partial, reduce
-from itertools import chain, groupby
+from itertools import groupby
 from pathlib import Path
 from typing import Annotated, Any, Callable, Literal, Union
```

Ruff (0.16.1)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)